proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -39,6 +39,13 @@ message TxResponse {
   // the timestamps of the valid votes in the block.LastCommit. For height == 1,
   // it's genesis time.
   string timestamp = 12;
+  // Events defines all the events emitted by processing a transaction. Note,
+  // these events include those emitted by processing all the messages and those
+  // emitted from the ante handler. Whereas Logs contains the events, with
+  // additional metadata, emitted only by processing the messages.
+  //
+  // Since: cosmos-sdk 0.45
```

Comment:
Since we're backporting:

```suggestion
  // Since: cosmos-sdk 0.42.11
```

Reply:
0.42.11 does not say anything about the availability in 0.44.x and 0.45.x.

`0.42.11, 0.44.5, 0.45` is the complete information needed.